compiler/include/expr.h

```diff
@@ -40,6 +40,8 @@ class Expr : public BaseAST {
   virtual bool    isStmt()                                           const;
   virtual QualifiedType qualType();
   virtual void    verify();
+          void    verify(AstTag expectedTag);
+          void    verifyParent(const Expr* child, const char* field);
```

Comment:
Could you put this verify/verifyParent change (also in expr.cpp) in a separate PR please?